README.md

```diff
@@ -117,9 +117,9 @@ function getUser(): User;
 
 **[⬆ back to top](#table-of-contents)**
 
-### Use searchable names
+### Use searchable names / Avoid Magic Literals
 
-We will read more code than we will ever write. It's important that the code we do write must be readable and searchable. By *not* naming variables that end up being meaningful for understanding our program, we hurt our readers. Make your names searchable. Tools like [ESLint](https://typescript-eslint.io/) can help identify unnamed constants.
+We will read more code than we will ever write. It's important that the code we do write must be readable and searchable. By *not* naming variables that end up being meaningful for understanding our program, we hurt our readers. Make your names searchable. Tools like [ESLint](https://typescript-eslint.io/) can help identify magic literals.
```

Comment:
The term _searchable names_ is just what it says and it's easy to understand for a beginner who's not yet comfortable with all of the programming dictionary. While you say _magic literals_, others call them [_magic strings_](https://deviq.com/antipatterns/magic-strings).  

I consider the term _searchable names_ less confusing and easier to understand for someone who is a beginner.  
I do appreciate your contribution, and I'd suggest that we add it to the end of the paragraph instead:

> ...can help identify unnamed constants (also known as _magic literals_ and _magic strings_).

Reply:
And in the end, the name _searchable names_ is referenced in the original _Clean Code_ book (see Chapter 2, page 22 - _Use Searchable Names_).

Reply:
Good point - the terminology for _magic literals_ referes to both _magic numbers_ and _magic strings_. Indeed though someone might be overwhelmed by this terminology. 

Addressed this as per the above comments, explicitly mentioning _magic numbers_ and _magic strings_.